linq.d.ts

```diff
@@ -33,7 +33,7 @@ declare namespace Enumerable {
   export function unfold<T>(seed: T, func: (value: T) => T): IEnumerable<T>;
   export function defer<T>(enumerableFactory: () => IEnumerable<T>): IEnumerable<T>;
 
-  export interface IEnumerable<T> {
+  export interface IEnumerable<T> extends Iterable<T> {
```

Comment:
Might be helpful: http://stackoverflow.com/questions/38968894/recommended-way-to-implement-iteratort-in-typescript-before-es6